app/helpers/reactjs_helper.rb

```diff
@@ -17,12 +18,16 @@ def react_component(name, props = {})
     content_tag('foreman-react-component', '', :name => name, :data => { props: props })
   end
 
+  def webpacked_plugins_with_css
```

Comment:
can we have the `global` here? :thinking: 
```suggestion
  def webpacked_plugins_with_global_css
```